src/DotNetEnv/Extensions/CreateDictionaryOption.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace DotNetEnv.Extensions
+{
+    public enum CreateDictionaryOption
+    {
+        /// <summary>
+        /// Default behaviour, throws on duplicates.
+        /// </summary>
+        Default,

```

Comment:
As above, please name this `Throw`, not default

Reply:
Renamed it, but changed ExtensionMethod Name too.

Reason is mentioned in the discussion on ToDictionary (https://github.com/tonerdo/dotnet-env/pull/88#discussion_r1431622773).